clang/test/Analysis/use-after-move.cpp

```diff
@@ -570,13 +570,8 @@ void differentBranchesTest(int i) {
   {
     A a;
     a.foo() > 0 ? a.foo() : A(std::move(a)).foo();
-#ifdef DFS
-    // peaceful-note@-2 {{Assuming the condition is false}}
-    // peaceful-note@-3 {{'?' condition is false}}
-#else
-    // peaceful-note@-5 {{Assuming the condition is true}}
-    // peaceful-note@-6 {{'?' condition is true}}
-#endif
```

Comment:
What is the intent behind this change? Is it just opportunistic cleanup, or is there a connection to the core goals of the commit?

Reply:
To be fair, I didn't really investigate. But I believe that probably the graph looks different, and it always picks the "true" path. So, this is a necessary change, but I didn't find this as a regression.

Reply:
I see, thanks for the explanation. I agree that this isn't an important question and simply adjusting the TC is the right solution.